automaxlair/matchup_scoring.py

```diff
@@ -128,6 +128,26 @@ def calculate_damage(
         modifier *= 0.75
     # Ignore weather for now
     # Ignore crits
+
+    # It needs to be before STAB
+    if move.type_id == 'normal':
+        if attacker.ability_name_id == 'refrigerate':
+            move.type_id = 'ice'
+            modifier *= 1.2
+        elif attacker.ability_name_id == 'aerilate':
+            move.type_id = 'fly'
+            modifier *= 1.2
+        elif attacker.ability_name_id == 'alvanize':
```

Comment:
I think the ability should be 'galvanize', though I don't think anything in SwSh has it.